source/stylesheets/refills/unstyled/_cards.scss

```diff
@@ -0,0 +1,20 @@
+.cards {
+  @include display(flex);
+  @include flex-wrap(wrap);
+  @include justify-content(space-between);
+
+  .card {
+    @include flex-basis(20em);
+    @include flex-grow(1);
+    margin: 0 1em 0 1em;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `0 1em` instead of `0 1em 0 1em`